content/blog/2013/2013-06-21-reducing-the-of-threads-in-jenkins-ssh-slaves.html

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Likewise here, can the filename be changed and replace it with a redirect somehow?